Automattic/jetpack
# JITMS: Adding jp-admin-notices IDs for the Boost plugin

Ideally there would be consistency in terms of JITM banner placement across all standalone Jetpack plugins, as well as within the Jetpack dashboard and My Jetpack.

This PR aimed to add more consistency where possible for the three standalones that did not have the 'jp-admin-notices' ID on their dashboard pages, however now just focuses on Boost.


#### Other information:


#### Jetpack product discussion

Related to this project: p9dueE-65G-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* If testing using the Beta plugin, make sure this branch is checked out for the VideoPress plugin within the Beta plugin dashboard
* Visit the Jetpack Boost admin page and confirm that the JITM is displayed as expected, which is not a full-screen banner at the top of the page (except Search) but a banner that fits the admin page content width. Check the page source to confirm it's being included via the 'jp-admin-notices' ID.

Expected Boost display:

<img width="1665" alt="Screen Shot 2022-11-16 at 4 34 02 pm" src="https://user-images.githubusercontent.com/16754605/202238871-1de1643c-8512-443b-bda9-7e2567b800ed.png">



With Search and VideoPress (originally included in this PR), these are somewhat more complex so I will leave those to the product teams as it is outside the scope of the project that initiated this.
